### PR TITLE
Change url for SuplaDevice library

### PR DESCRIPTION
**Pull request by klew**
We decided to move SuplaDevice library to another repository with different url. Old one will not be maintained anymore.

### CODE REVIEW

**Comment by per1234** [comment]
Hello @klew. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/1636#issuecomment-1204114505

**Review by umbynos — APPROVED** [review]

**Pull request merged**
